bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="frequency">
+		<item-type>Number:Frequency</item-type>
+		<label>Electric Frequency</label>
+		<description>Electric frequency of the input/output voltage of the inverter</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Frequency</tag>
+		</tags>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="battery-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Battery Temperature</label>
+		<description>Battery Temperature</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="last-retrieve-time-stamp">
+		<item-type>DateTime</item-type>
+		<label>Last Retrieve Time Stamp</label>
+		<description>Last time with a successful retrieval of data</description>
+		<category>Time</category>
+		<state pattern="yyyy-MM-dd HH:mm:ss" readOnly="true"/>
+	</channel-type>
+	<channel-type id="raw-data-type" advanced="true">
+		<item-type>String</item-type>
+		<label>Raw Data</label>
+		<description>Raw data retrieved from the system as a JSON</description>
```

Comment:
The removed channel override had this version which I think was your latest revision:
```suggestion
		<description>The raw JSON data retrieved from the inverter's Wi-Fi module.</description>
```